input/blog/2020-12-10-cake-vscode-extension-release-1.0.0.md

```diff
@@ -0,0 +1,36 @@
+---
+title: Cake VSCode Extension Release 1.0.0
+category: News
+author: gep13
+---
+
+The Cake Team are very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!
```

Comment:
Shouldn't it be `The Cake Team is...`? 

```suggestion
The Cake Team is very happy to announce the 1.0.0 release of the [Cake Extension for VSCode](https://marketplace.visualstudio.com/items?itemName=cake-build.cake-vscode).  This release has been a long time in the making, but we are hoping that it will be worth the wait!
```